mitodl/mit-open
# contrast fix for banner on channel pages

### What are the relevant tickets?

Closes [#1045](https://github.com/mitodl/mit-open/issues/1045)


### Description (What does it do?)
Adds a 30% opacity overlay to the channel pages so the text is more visible

This PR also includes changes to some banner backgrounds that are flipped horizontally to match the figma.

### Screenshots (if appropriate):
Before:
<img width="1371" alt="Screenshot 2024-06-11 at 9 02 39 AM" src="https://github.com/mitodl/mit-open/assets/196425/046b82d5-bb61-4bdf-b464-400b74087396">


After
<img width="1367" alt="Screenshot 2024-06-11 at 9 02 32 AM" src="https://github.com/mitodl/mit-open/assets/196425/3bc306e7-b574-4635-9a09-6234b6490ea5">


### How can this be tested?
1. Checkout this branch. 
2. ensure that the banner has an overlay that makes the text more visible and matches the figma (linked in the ticket)
